breaking-front/src/constants/path.js

```diff
@@ -14,6 +14,7 @@ export const PAGE_PATH = {
   SEARCH_POST: `/search/post`,
   SEARCH_HASHTAG: `/search/hashtag`,
   SEARCH_USER: `/search/user`,
+  BREAKING_MISSION_MAP: 'breaking_mission/map',
```

Comment:
URL에는 언더바( _ )보다 대쉬( - )를 권장한다고 합니다!
```suggestion
  BREAKING_MISSION_MAP: 'breaking-mission/map',
```
https://developers.google.com/search/docs/advanced/guidelines/url-structure?hl=ko&visit_id=637979746999342167-1757659139&rd=1